packages/commons/src/events/EventMetadata.ts

```diff
@@ -32,7 +32,16 @@ export const EventStatus = z.enum([
 export type EventStatus = z.infer<typeof EventStatus>
 
 export const CustomFlags = {
-  CERTIFICATE_PRINTED: 'certificate-printed'
+  CERTIFICATE_PRINTED: 'certificate-printed',
+  DECLARATION_INCOMPLETE: 'declaration-incomplete',
+  DECLARATION_REQUIRES_UPDATES: 'declaration-requires-updates',
+  DECLARATION_PENDING_EXTERNAL_VALIDATION:
+    'declaration-pending-external-validation',
+  REGISTRATION_CERTIFY_NEW_REGISTRATION:
+    'registration-certify-new-registration',
+  REGISTRATION_RE_CERTIFY_AFTER_CORRECTION:
+    'registration-re-certify-after-correction',
+  REGISTRATION_CORRECTION_REQUESTED: 'registration-correction-requested'
```

Comment:
We need to verify whether we really need all these flags

Reply:
@jamil314 let's keep the flag strings super simple and short so and make them more explicit later if needed.

printed
incomplete
requires-updates
correction-requested